scenario/ArbitrumBridgeScenario.ts

```diff
@@ -0,0 +1,116 @@
+import { scenario } from './context/CometContext';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { matchesDeployment, createCrossChainProposal } from './utils';
+import { calldata } from '../src/deploy';
+import { ArbitrumBridgeReceiver } from '../build/types';
+
+scenario(
+  'upgrade Arbitrum governance contracts and ensure they work properly',
```

Comment:
FYI, we have a `CrossChainGovernanceScenario.ts` file now. Up to you if you want to move it there or keep it separate.

Reply:
Done